langchain/src/chains/combine_docs_chain.ts

```diff
@@ -96,7 +96,7 @@ export class StuffDocumentsChain
 export interface MapReduceDocumentsChainInput extends StuffDocumentsChainInput {
   maxTokens: number;
   maxIterations: number;
-  combineDocumentsChain: BaseChain;
+  combineDocumentChain: BaseChain;
```

Comment:
This is technically a breaking change so we'd want to keep around both old and new names